ee/modules/502-delivery/templates/argocd/server/argocd-server-rolebinding.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Roles must be placed to `rbac-for-us`/`rbac-to-us` files only (checked by linter).

Reply:
Ok. Sadly, this will break the grouping by components.